calico_containers/tests/st/test_mainline.py

```diff
@@ -10,17 +10,17 @@ def run_mainline(self, ip1, ip2):
         Setup two endpoints on one host and check connectivity.
         """
         host = DockerHost('host')
-        host.start_etcd()
 
-        host_ip = docker.inspect("--format", "'{{ .NetworkSettings.IPAddress }}'", host.name).stdout.rstrip()
-        etcd_port = "ETCD_AUTHORITY=%s:2379" % host_ip
+        # etcd_ip = docker.inspect("--format", "{{ .NetworkSettings.IPAddress }}", "etcd").stdout.rstrip()
+        etcd_ip = self.ip
+        etcd_port = "ETCD_AUTHORITY=%s:2379" % etcd_ip
```

Comment:
This naming is confusing.  The etcd "port" is 2379.  The combination of a host and port is called an "authority."